gradle.properties

```diff
@@ -7,10 +7,10 @@ cordaProductVersion = 5.0.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 112
+cordaApiRevision = KOTLIN17
```

Comment:
Surely this cannot be right...

Reply:
It's WIP, so that `corda-runtime-os` can find artifacts to compile against :wink:.

Reply:
Now updated to 113 :grinning: